### DISCUSSION TITLE
Env variables

### DISCUSSION THREAD

**Discussion by ilbarone87** [dani-garcia/vaultwarden#2980] [Q&A] [answered]
Hello all,
Can the env variables like ADMIN_TOKEN or SMTP_PASSWORD be read from a .txt file (or any other kind of file)?

Thanks

**Comment by BlackDex** [comment, marked as answer]
See: https://github.com/dani-garcia/vaultwarden/discussions/2759

**Comment by ilbarone87** [comment]
Thanks
